backend/casemanagement/src/main/java/org/pucar/dristi/service/CaseManagerService.java

```diff
@@ -174,4 +177,14 @@ public JSONArray constructJsonArray(String json, String jsonPath) {
 			throw new CustomException("JSON_ARRAY_CONSTRUCTION_ERR", e.getMessage());
 		}
 	}
+
+	public List<CaseSummary> getCaseSummary(CaseRequest caseRequest) {
+		try {
```

Comment:
implement validation if no criteria fields are provided then return the empty list.